Pod/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -528,6 +532,17 @@ - (void)applicationWillTerminate
     }];
 }
 
+-(void)applicationDidFinishLaunching
```

Comment:
Although if that was async we'd have to be careful how it gets loaded from disk in the accessor so we do not run into race conditions.